packages/wallet-api-manifest-validator/cli/cli.ts

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env node
+import { Command, Option, runExit } from "clipanion";
+import * as fs from "fs";
+import path from "path";
+import { validateManifest } from "../src/validator";
+
+// eslint-disable-next-line @typescript-eslint/no-floating-promises
+runExit(
+  class ValidateFile extends Command {
+    static paths = [[`validate -h |`], Command.Default];
+
+    fileOrDir = Option.String();
+
+    details = Option.Boolean(`--details, -details`);
+
+    enableState = Option.Boolean(`--enableState, -enableState`);
+
+    throwError = Option.Boolean(`--throwError, -throwError`);
```

Comment:
For a CLI application, it's good practice to follow the [POSIX Conventions for Command Line Arguments](http://www.iitk.ac.in/esc101/05Aug/tutorial/essential/attributes/_posix.html) and the [GNU Standards for Command Line Interfaces](https://www.gnu.org/prep/standards/html_node/Command_002dLine-Interfaces.html) extension that states that the single hyphen (`-`) should be used for short hand option (single alphanumeric character).
The double hyphens form (`--`) should be used for verbose (usually full word) version of the option.

```suggestion
    details = Option.Boolean(`--details, -d`);

    enableState = Option.Boolean(`--enableState, -e`);

    throwError = Option.Boolean(`--throwError, -t`);
```

Reply:
I didn't saw it on the sources but, Is it mandatory for the short one to be the first char of the full word option ? 
eg (--throwError, -e) could be ok with the conventions ?